monai/transforms/compose.py

```diff
@@ -168,3 +168,64 @@ def inverse(self, data):
         for t in reversed(invertible_transforms):
             data = apply_transform(t.inverse, data, self.map_items, self.unpack_items)
         return data
+
+
+class OneOf(Compose):
+    """
+    ``OneOf`` provides the ability to radomly choose one transform out of a
+    list of callables with predfined probabilities for each.
+
+    Args:
+        transforms: sequence of callables.
+        weights: probabilities corresponding to each callable in transforms.
+            Probabilities are normalized to sum to one.
+
+    OneOf inherits from Compose and uses args map_items and unpack_items in
+    the same way.
+    """
+
+    def __init__(
+        self,
+        transforms: Optional[Sequence[Callable]] = None,
+        weights: Optional[Sequence[float]] = None,
+        map_items: bool = True,
+        unpack_items: bool = False,
+    ) -> None:
+        if transforms is None:
+            transforms = []
```

Comment:
Would there be any utility in having an empty list of transforms? I would have thought it would be best to throw an error at this point.

Reply:
I see that you've copied this from the `Compose` constructor. In which case, better to use `super().__init__()`.

Reply:
Hi @rijobro , thanks for the helpful comments. I just made a new commit that uses `super().__init__()` which should clean things up hopefully